test/rules/assertions/checkParamNames.js

```diff
@@ -266,20 +283,76 @@ export default {
           /**
            * @param cfg
            * @param cfg.foo
-           * @param [cfg.foo="with a default"]
```

Comment:
I assume this was due to our previously silently ignoring destructured. Is there a rule elsewhere checking duplicate destructured param names? I think that should be reported if not.

Reply:
This one was a preemptive removal based on the previous examples, but after testing I see that `cfg.foo` and `[cfg.foo="with a default"]` are treated as a single property.